### ISSUE TITLE
Add Classic Bomb Arrow Timer Back

### ISSUE THREAD

**Issue by Shadowblitz16** [transhumandesign/kag-base#631]
Please bring back classic bomb arrow timer.
I have been playing Kag for 7 years and since classic I have never been able to bomb jump like I used to.

I propose classic's bomb arrow timer be brought back for not only for my sake but for new players, and players that cannot multitask very well.

learning bomb jumping is one thing but actually using it in the heat of battle is another

if people don't like the users with the ability to see the location of their bombs then it should only work the bomb your holding.

I never understood why this was removed, I guess a lot of people just didn't like users having the ability to learn easier idk.

**Comment by eps0003** [comment]
Imagine wanting to change something just because you're bad at it lol

**Comment by MrPotatobird** [comment]
I mean, everyone was pretty fond of the cursor indicating slash timing. Is this that different?
As he said, it's even more for the sake of new players / baddies.

**Comment by bunniewormy** [comment]
Agreed, the game is already fucking hard for newbies, we should at least make some parts of it easier

**Comment by Shadowblitz16** [comment]
I think the ability to disable the bomb timer should be in the graphics options.
just in case pro players feel that it's in the way.

**Comment by Panky123** [comment]
Imagine being a dev and insulting someone because they want to help new players

**Comment by Harrison-Miller** [comment]
Having the option to disable the bomb indicator would be a good compromise, I think this is fine to add to the game especially if it's optional. It should probably only be on bombs you've thrown though.

**Comment by Shadowblitz16** [comment]
> Having the option to disable the bomb indicator would be a good compromise, I think this is fine to add to the game especially if it's optional. It should probably only be on bombs you've thrown though.
Or in your hand.
Its can't help you learn bomb jumping if its only for thrown bombs
